data/ecosystems/n/noir-lang.toml

```diff
@@ -143,3 +143,225 @@ url = "https://github.com/vezenovm/simple_shield"
+
+[[repo]]
+url = "https://github.com/noir-playground/noir-playground"
+
+[[repo]]
+url = "https://github.com/visoftsolutions/noir_rs"
+
+[[repo]]
+url = "https://github.com/visoftsolutions/noir_java"
+
+[[repo]]
+url = "https://github.com/visoftsolutions/noir_swift"
+
+[[repo]]
+url = "https://github.com/hooperben/noir-hardhat-template"
+
+[[repo]]
+url = "https://github.com/iam-robi/noir-starter-nuxt"
+
+[[repo]]
+url = "https://github.com/HaMMeRSI/noir-vite-starter"
+
+[[repo]]
+url = "https://github.com/noir-lang/noir/tree/master/noir_stdlib"
+
+[[repo]]
+url = "https://github.com/zkworks-xyz/noir-base64"
+
+[[repo]]
+url = "https://github.com/doctoruber/complexnr"
+
+[[repo]]
+url = "https://github.com/doctoruber/noir-fixed-point"
+
+[[repo]]
+url = "https://github.com/doctoruber/statnr"
+
+[[repo]]
+url = "https://github.com/madztheo/noir-date"
+
+[[repo]]
+url = "https://github.com/shuklaayush/noir-bigint"
+
+[[repo]]
+url = "https://github.com/resurgencelabs/signed_int"
+
+[[repo]]
+url = "https://github.com/resurgencelabs/fraction"
+
+[[repo]]
+url = "https://github.com/colinnielsen/noir-u2b/tree/main"
+
+[[repo]]
+url = "https://github.com/0x3327/ZKFloat"
+
+[[repo]]
+url = "https://github.com/vocdoni/smtverifier-noir/tree/main"
+
+[[repo]]
+url = "https://github.com/jordan-public/zk-optimized-sparse-merkle-tree"
+
+[[repo]]
+url = "https://github.com/privacy-scaling-explorations/zk-kit/tree/main/packages/circuits/noir"
+
+[[repo]]
+url = "https://github.com/privacy-scaling-explorations/zk-kit/tree/main/packages/smt"
+
+[[repo]]
+url = "https://github.com/tomoima525/noir-merkle-root"
+
+[[repo]]
+url = "https://github.com/storswiftlabs/quantized_arithmetic"
+
+[[repo]]
+url = "https://github.com/michaelelliot/noir-sha1"
+
+[[repo]]
+url = "https://github.com/michaelelliot/noir-sha2"
+
+[[repo]]
+url = "https://github.com/TaceoLabs/noir-hydra"
+
+[[repo]]
+url = "https://github.com/TaceoLabs/noir-griffin"
+
+[[repo]]
+url = "https://github.com/TaceoLabs/noir-aes"
+
+[[repo]]
+url = "https://github.com/jat9292/noir-elgamal"
+
+[[repo]]
+url = "https://github.com/onurinanc/noir-bls-signature"
+
+[[repo]]
+url = "https://github.com/aragonzkresearch/noir-trie-proofs"
+
+[[repo]]
+url = "https://github.com/0x3327/skproof"
+
+[[repo]]
+url = "https://github.com/metavind/noir-ml"
+
+[[repo]]
+url = "https://github.com/storswiftlabs/zkml-noir"
+
+[[repo]]
+url = "https://github.com/storswiftlabs/matrix_operations"
+
+[[repo]]
+url = "https://github.com/storswiftlabs/convolution"
+
+[[repo]]
+url = "https://github.com/sequi-xyz"
+
+[[repo]]
+url = "https://github.com/TalDerei/cyclone"
+
+[[repo]]
+url = "https://github.com/porco-rosso-j/safe-recovery-noir"
+
+[[repo]]
+url = "https://github.com/ruizehung/Zero-Knowledge-Sudoku-Wordle-Trivia"
+
+[[repo]]
+url = "https://github.com/signorecello/zcaptcha"
+
+[[repo]]
+url = "https://github.com/resurgencelabs/hangman"
+
+[[repo]]
+url = "https://github.com/MeloCafe"
```

Comment:
this is not a repo URL, it looks like an org URL, either remove or move up to the organizations list at the beginning of the file if the project is using Noir